beacon-chain/blockchain/kzg/kzg.go

```diff
@@ -33,11 +31,15 @@ type Bytes32 = ckzg4844.Bytes32
 
 // CellsAndProofs represents the Cells and Proofs corresponding to
 // a single blob.
+
 type CellsAndProofs struct {
 	Cells  []Cell
 	Proofs []Proof
 }
 
+// Set to false to not change any existing behavior
```

Comment:
Currently using this to toggle between go-eth-kzg and c-kzg cc @nisdas for visibility

Reply:
Justin pointed out that this is a build time flag -- it should be a cli flag, but I just didn't know how to do that at the time and went with the easiest option initially